src/porepy/applications/verification_setups/terzaghi_biot.py

```diff
@@ -46,7 +46,7 @@
 import porepy.models.fluid_mass_balance as mass
 import porepy.models.poromechanics as poromechanics
 from porepy.applications.classic_models.biot import BiotPoromechanics
```

Comment:
I suggest renaming from classic_models to derived_models.

Reply:
Done.